src/Docblock.php

```diff
@@ -308,86 +339,87 @@ public function getTags($tagName = null) {
 	 * 
 	 * @return ArrayList
 	 */
-	public function getSortedTags() {
+	public function getSortedTags(): ArrayList {
 		if ($this->comparator === null) {
-			$this->comparator = new TagNameComparator(); 
+			$this->comparator = new TagNameComparator();
 		}
 
 		// 1) group by tag name
 		$group = new Map();
-		foreach ($this->tags as $tag) {
+		foreach ($this->tags->toArray() as $tag) {
```

Comment:
`->toArray()` not needed afair, you can still iterate over the `ArrayList`

Reply:
Yes, it runs correctly. But we marked `ArrayList::current()` as `@internal`  so `psalm` raises an error when someone access to an internal method.
We can update `ArrayList` or suppress this error in this file.